lang/elixir/elixir.talon

```diff
@@ -0,0 +1,47 @@
+code.language: elixir
+-
+tag(): user.code_functional
+tag(): user.code_concurrent
+
+tag(): user.code_comment_line
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_functions
+tag(): user.code_keywords
+tag(): user.code_libraries
+tag(): user.code_operators_array
+tag(): user.code_operators_assignment
+tag(): user.code_operators_math
+tag(): user.code_operators_lambda
+
+settings():
+    user.code_private_function_formatter = "snake_case"
+    user.code_public_function_formatter = "snake_case"
+    user.code_private_variable_formatter = "snake_case"
+    user.code_public_variable_formatter = "snake_case"
+
+# Elixir-specific grammars
+state def: "def "
+state defp: "defp "
+state if: "if "
+state else: "else"
+state case: "case "
+state cond: "cond do"
+state try: "try do"
+state rescue: "rescue"
+state after: "after"
+state end: "end"
```

Comment:
I could swear we had a system for making these a list, but inspecting Python javascript in java, they all seem to put these up in the Talon file raw 🤷‍♂️. It might be nice to do that but it can be a followup